CHANGELOG.md

```diff
@@ -1,12 +1,61 @@
 1.12.0 (in progress)
 ====================
 
+Deprecations
+------------
+
+The following `1.x` APIs/events are now deprecated and scheduled for removal
+in `twilio-video.js@2.0.0`:
+
+- Participant's "trackAdded" and "trackRemoved" events
```

Comment:
Nice, btw, I think we need to update the usage example in README.md?

Reply:
@markandrus WDYT about updating README.md in the 2.x branch? I think we can do it once and for all there.